### ISSUE TITLE
[dispatcher] Allow eventtype to be numeric or symbolic for ease-of-use

### ISSUE THREAD

**Issue by gkistler** [Clam-/pyBurlyBot#63]
Straightforward, `util.dispatcher.dispatch` is the relevant function for implementation.  I've got this partially implemented, just need to thoroughly test then I'll push.

Creating issue so I don't forget.

NOTE: this will also allow numeric/symbolic IRC event ambiguity in the `util.container.send_and_wait` parameters `interestede` and `stope`, which is nice.

**Comment by Clam-** [comment]
Oh huh, I thought numeric events already worked:
https://github.com/Clam-/pyBurlyBot/blob/master/util/client.py#L438

I never actually tested this though I don't think.

**Comment by gkistler** [comment]
Resolved in a67d90e9070991107427711c542d78440a580ebb

I misunderstood the issue a bit, `symbolic_to_numeric` lookup was already being performed for dispatching.  I just needed to add the reverse to ensure when the server sends a numeric event the corresponding symbolic event is also dispatched (when it's in our `numeric_to_symbolic` dict).

The referenced commit also makes sure things are case insensitive, in case any IRCDs send lowercase symbolic events (unsure if that's actually a thing).
